src/static/containers/Home/index.js

```diff
@@ -54,5 +71,11 @@ const mapStateToProps = (state) => {
     };
 };
 
-export default connect(mapStateToProps)(HomeView);
+const mapDispatchToProps = (dispatch) => {
+    return {
+        actions: bindActionCreators(actionCreators, dispatch)
```

Comment:
if we choose to go with my approach. you need to put a proper name, instead of 'actions'